src/sparseml/export/export.py

```diff
@@ -207,16 +196,29 @@ def export(
         source_path=source_path,
         target_path=target_path,
         deployment_directory_name=deployment_directory_name,
-        deployment_directory_files=helper_functions.deployment_directory_structure,
+        deployment_directory_files_mandatory=helper_functions.deployment_directory_files_mandatory,  # noqa: E501
+        deployment_directory_files_optional=helper_functions.deployment_directory_files_optional,  # noqa: E501
         onnx_model_name=onnx_model_name,
     )
+
+    _LOGGER.info(
+        f"Applying optimizations: {graph_optimizations} to the exported model..."
+    )
+    apply_optimizations(
```

Comment:
`graph_optimization` contains a mapping that specifies all the possible optimizations for a specific integration. the `apply_optimization` is a general function that establishes which of those optimizations should be actually applied and applies them. Why worried about onnx specificity? We could always pass the `deployment_target` as an argument of `apply_optimizations` - this actually demonstrates the usefulness of this abstraction.

Reply:
I still fail to see how folding this function into each of the `IntegrationHelperFunctions` is beneficial. It would add to code duplication, and IMO worsen the readability for the user. I'd propose to leave it for now, it's a very small component, can be refactored easily if needed.